specification/desktopvirtualization/resource-manager/readme.java.md

```diff
@@ -16,18 +16,19 @@ output-folder: $(azure-libraries-for-java-folder)/azure-mgmt-desktopvirtualizati
 
 ``` yaml $(java) && $(multiapi)
 batch:
-  - tag: package-2019-01-23-preview
+  - tag: package-2019-12-10-preview
 ```
 
-### Tag: package-2019-01-23-preview and java
+### Tag: package-2019-12-10-preview and java
 
-These settings apply only when `--tag=package-2019-01-23-preview --java` is specified on the command line.
+These settings apply only when `--tag=package-2019-12-10-preview --java` is specified on the command line.
 Please also specify `--azure-libraries-for-java=<path to the root directory of your azure-sdk-for-java clone>`.
 
-``` yaml $(tag) == 'package-2019-01-23-preview' && $(java) && $(multiapi)
+``` yaml $(tag) == 'package-2019-12-10-preview' && $(java) && $(multiapi)
 java:
   namespace: com.microsoft.azure.management.desktopvirtualization.v2019_01_23_preview

```

Comment:
namespace and output-folder: version should be updated to 2019-12-10.

Reply:
making changes.